pandas/tests/io/pytables/test_store.py

```diff
@@ -296,6 +296,61 @@ def test_keys(self, setup_path):
             assert set(store.keys()) == expected
             assert set(store) == expected
 
+    def test_no_track_times(self, setup_path):
+
+        # GH 32682
+        # enables to set track_times (see `pytables` `create_table` documentation)
+
+        import hashlib
```

Comment:
all imports should be at the top; if you need to skip for a version you can do it with a `@td.skip_if_no`. add a test for >= 3.4.3 and one fore before that.

Reply:
done :+1: